### PR TITLE
Dockerise epic

### LINKED ISSUE #94: Dockerise extract_epic.py
##Description
Write a Dockerfile for `extract_epic.py`

##Required Files
- `extract_epic.py`
- `Dockerfile`
##User Story
As an engineer I need my scripts to be run regularly and serverless so I can continually be collecting more data.

### PR DESCRIPTION
**Pull request by shindym**
- Lambda function made 
- `Dockerfile` made
- Lamdba function tested locally 
Closes #94 #81

### CODE REVIEW

File: pipeline/extract/epic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM public.ecr.aws/lambda/python:3.12
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+# Copy requirements.txt
```
Comment (hhm970) [inline pipeline/extract/epic/Dockerfile:5]: Make sure to get rid of commented-out lines

**Review by hhm970 — DISMISSED** [review]
Honestly it looks okay - my comment on your Dockerfile was purely me nitpicking!

File: pipeline/extract/epic/Dockerfile
Comment (hhm970) [inline pipeline/extract/epic/Dockerfile:1]: Dockerfile is nicely formatted

**Review by hhm970 — APPROVED** [review]
Looks good to me!

**Review by Eibironke — APPROVED** [review]
👍

**Pull request merged**